traveller/modules/schedule/view.py

```diff
@@ -50,7 +64,9 @@ def add_activity(year, day_id, act_type):
     if act_type == 'normal_activity':
         day = Day.query.get(day_id)
         form = NormalActivityForm()
-        form.validate()
```

Comment:
I do not think you can just delete `form.validiate` as there is more data than just the start and the end time.